stderr of test 'tst302` in directory 'monetdb5/mal` itself:


# 09:00:01 >  
# 09:00:01 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=30157" "--set" "mapi_usock=/var/tmp/mtest-21311/.s.monetdb.30157" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/export/scratch1/mk/default//Linux/var/MonetDB/mTests_monetdb5_mal" "--set" "mal_listing=2" "--set" "embedded_r=yes"
# 09:00:01 >  

# builtin opt 	gdk_dbpath = /export/scratch1/mk/default//Linux/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 30157
# cmdline opt 	mapi_usock = /var/tmp/mtest-21311/.s.monetdb.30157
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	gdk_dbpath = /export/scratch1/mk/default//Linux/var/MonetDB/mTests_monetdb5_mal
# cmdline opt 	mal_listing = 2
# cmdline opt 	embedded_r = yes
# cmdline opt 	gdk_debug = 536870922

# 14:00:49 >  
# 14:00:49 >  "mclient" "-lmal" "-ftest" "-Eutf-8" "--host=/var/tmp/mtest-20510" "--port=36088"
# 14:00:49 >  

MAPI  = (monetdb) /var/tmp/mtest-20510/.s.monetdb.36088
QUERY = #propagate INT exception
        # (you should be able to pass it upwards though)
        function foo();
        raise	PermissionDeniedException := 1234;
        	io.print("should not come here\n");
        end;
        mdb.List("user","foo");
        user.foo();
ERROR = !MALException:user.foo[4]:Exception not caught



# 14:00:49 >  
# 14:00:49 >  "Done."
# 14:00:49 >  

